macOS/LocalPackages/DataBrokerProtection/Sources/DataBrokerProtection/Shared/Operations/DataBrokerOperation.swift

```diff
@@ -64,7 +64,7 @@ class DataBrokerOperation: Operation, @unchecked Sendable {
     private var _isFinished = false
 
     deinit {
-        Logger.dataBrokerProtection.log("Deinit operation: \(String(describing: self.id.uuidString), privacy: .public)")
+        Logger.dataBrokerProtection.log("Deinit DataBrokerOperation: \(String(describing: self.id.uuidString), privacy: .public)")
```

Comment:
Similar to the notification point below, updating some logs to make it clearer to me what's going on.